man/fi_cxi.7.md

```diff
@@ -1314,6 +1314,25 @@ The CXI provider checks for the following environment variables:
 :   Enable enforcement of triggered operation limit. Doing this can prevent
     fi_control(FI_QUEUE_WORK) deadlocking at the cost of performance.
 
+*FI_CXI_ENABLE_WRITEDATA*
+:   Controls provider support for the fi_writedata() and fi_inject_writedata()
+    RMA operations. When enabled, the CXI provider implements handling to
+    generate solicited RMA completions that include immediate data; completions
+    will include FI_REMOTE_CQ_DATA and will report source information when
```

Comment:
Consider changing the the above sentence to: When enabled and the domain attribute cq_data_size is non-zero, the CXI provider implements...

I realize you clarify this in the next paragraph, but I think it helps to make it apparent early we are not circumventing the libfabric method for indication, just selective enabling that for performance/resource reasons since there is not a unique cq_data_size for RMA and MSG/TAGGED.